tests/util.py

```diff
@@ -6,7 +6,21 @@
 
 def create_test_user(slack_id):
     """
-    Create a test user with slack id, and with all other attributes the same.
+    Create a test user with slack id, and with all other attributes set.
+
+    ==========  =============================
+    Property    Preset
+    ==========  =============================
+    Slack ID    ``slack_id``
+    Bio         I like puppies and kittens!
+    Email       admin@ubc.ca
+    Name        Iemann Atmin
+    Github      kibbles
+    Image URL   https://google.ca
```

Comment:
nitpick: maybe find an actual image URL, not just any URL. Might be useful if we actually want to test future modules that require retrieving an actual image.